spharpy/samplings/samplings.py

```diff
@@ -549,19 +555,18 @@ def gaussian(n_points=None, n_max=None, radius=1.):
         raise ValueError(
             "Either the n_points or n_max needs to be specified.")
 
+    if n_points is not None and (np.asarray(n_points).size > 1):
```

Comment:
shouldn't we also check whether it is an integer? At the moment there would be no error for floats